webviz_subsurface_components/py_expression_eval.py

```diff
@@ -0,0 +1,483 @@
+"""
+Copyright (c) 2021- Equinor ASA
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+
+This is a modification of py-expression-eval created by AxiaCore. 
+
+Moifications are done to obtain simplifed functionality for itended usage.
+
+Thanks to AxiaCore!
+
+`py-expression-eval:`
+
+Author: AxiaCore S.A.S. http://axiacore.com
+GitHub: https://github.com/AxiaCore/py-expression-eval/
+
+Based on js-expression-eval, by Matthew Crumley (email@matthewcrumley.com, http://silentmatt.com/)
+https://github.com/silentmatt/js-expression-eval
+
+Ported to Python and modified by Vera Mazhuga (ctrl-alt-delete@live.com, http://vero4ka.info/)
+
+You are free to use and modify this code in anyway you find useful. Please leave this comment
+in the code to acknowledge its original source. If you feel like it, I enjoy hearing about
+projects that use my code, but don't feel like you have to let me know or ask permission.
+"""
+# pylint: skip-file
+
+from __future__ import division
```

Comment:
```suggestion
```
We are on Python 3, so don't need `from __future__ import division`. 

Also, instead of `pylint` skipping the whole file, lets instead fix the easy parts and disable specific warnings?